easybuild/easyconfigs/m/miller/miller-6.4.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'GoPackage'
+
+name = 'miller'
+version = '6.4.0'
+modulename = 'github.com/johnkerl/miller/cmd/mlr'
+
+homepage = 'https://miller.readthedocs.io/'
+description = """Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/johnkerl/miller/releases/download/v%(version)s']
+sources = ['%(name)s-%(version)s.tar.gz']
```

Comment:
```suggestion
sources = [SOURCELOWER_TAR_GZ]
```

Reply:
There are a couple of tarballs available at the repository and I managed to build and test with both of them, but not sure about the differences. I choose the smallest one, but maybe that name will have to be modified in the future.
But for now, it seems OK :-)

Reply:
this is just a macro that expands to the pattern you just typed out (but with lowercase) which is needed now after the case change